FiguresInheritance/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FiguresInheritance.domain;
+using FiguresInheritance.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresInheritance
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // Единственное, нет проверок на отрицательные числа и на стороны (потипу как для треугольника)
+            // Надеюсь, это не проблема
+            var figuresContainer = new FiguresContainer();
```

Comment:
```suggestion
            var figuresContainer = new List<Figure>();
```
And so on and so forth